apps/nextra/pages/en/build/move/_meta.tsx

```diff
@@ -0,0 +1,18 @@
+export default {
+  index: {
+    title: "Introduction",
+  },
+  book: {
+    title: "Move Book",
+    theme: {
+      toc: false,
+      layout: "full",
```

Comment:
Can we remove `layout: "full"` and `toc: false` here? Unfortunately when you do that you lose  the breadcrumb at the top of each page ([example](https://developer-docs-nextra-git-greg-migrate-nextra-aptoslabs.vercel.app/en/build/move/book/SUMMARY)). 

I looked at each page in the Move book, I think the only ones that would benefit from full layout are the ones with wider columns. I've noted each one below. 

Places to apply layout full
1. Vector -> https://developer-docs-nextra-git-greg-migrate-nextra-aptoslabs.vercel.app/en/build/move/book/vector
2. References -> https://developer-docs-nextra-git-greg-migrate-nextra-aptoslabs.vercel.app/en/build/move/book/references
3. Tuples and Unit -> https://developer-docs-nextra-git-greg-migrate-nextra-aptoslabs.vercel.app/en/build/move/book/tuples